include/gridtools/stencil/frontend/cartesian/accessor.hpp

```diff
@@ -108,6 +107,14 @@ namespace gridtools {
                     return i >= Extent::iminus::value && i <= Extent::iplus::value && j >= Extent::jminus::value &&
                            j <= Extent::jplus::value && k >= Extent::kminus::value && k <= Extent::kplus::value;
                 }
+
+                template <class... Ts>
+                GT_FUNCTION GT_CONSTEXPR bool all(Ts... args) {
```

Comment:
I think I saw this snippet somewhere else. Was it exactly the same functionality and should it be re-used? Was storage related.

Reply:
This is the snippet that I meant https://github.com/GridTools/gridtools/commit/b96b672f4822edd949ee9081c42eb718a9033327

Reply:
I failed to find the same pattern. Maybe it was gone already.

Reply:
no action required, I'd say